recipes/fastcache/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "fastcache" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "867005774917f999854ffd303fbe3793c30c2dc9279b78ec315dd4e57e4284e6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+  run:
+    - python
+
+test:
+  imports:
+    - fastcache
+    - fastcache._lrucache
+
+  commands:
+    - python -c "import fastcache; fastcache.test()"
+
+  requires:
+    - pytest
+
+about:
+  home: https://github.com/pbrady/fastcache
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Please raise an issue/PR upstream to add a `MANIFEST.in` with at least the line `include LICENSE`. Also please xref that issue/PR back here. Thanks.